src/azure-cli/azure/cli/command_modules/rdbms/_help.py

```diff
@@ -689,6 +689,26 @@
 short-summary: List the private link resources supported for a MySQL server.
 """
 
+helps['mysql server encryption key'] = """
```

Comment:
Where is the command registration for `encyption key`? I only found `server key` related commands. Is it a typo?

Reply:
Fixed